draft-ietf-tsvwg-careful-resume.xml

```diff
@@ -553,7 +556,8 @@ Examples of the transitions between phases are provided in <xref target="Example
             <t>Validating Phase (Check flight_size on entry):
             On entry to the Validating Phase, if the flight_size is less
 	    equal to the PipeSize, the Normal phase is entered and CWND
```

Comment:
Is congestion_window_limited the correct trigger for moving to Normal phase here? Looks like we're describing the application limited case.